src/systems/filecoin_mining/sector/lifecycle.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Sector Lifecycle
+---
+
+In Filecoin, miners contribute storage capacity to the network in units of Sectors. These sectors work similar to real-life shipping containers; they are used provide a unique ID for storage / retrieval processes as well as ensuring the datas 'dimensions' conform with all other Sectors in the network.
+
+#### Sector creation
+
+At creation, a sectors space (raw-byte power or sector size) and time (duration) are defined. The defined space will determine the total sum of the sectors pledge collateral in FIL. The new sector may contain deals (either Deals or VerifiedDeals), Committed Capacity or a mixture of both. 
+
+The sector is then assigned a `SectorQuality`. which determines its Quality-Adjusted Power in the network, or consensus power (weight).
+
+`SectorQuality` is determined through a weighted average of multipliers, based on their contents:
+
+* Sectors full of `VerifiedDeals` will have a `SectorQuality` of `VerifiedDealWeightMultiplier/BaseMultiplier`.
+* Sectors full of regular `Deals` will have a `SectorQuality` of `DealWeightMultiplier/BaseMultiplier`.
+* Sectors with neither will have a `SectorQuality` of `BaseMultiplier/BaseMultiplier`.
+
+Once `SectorQuality` has been assigned, the Sector is now ready for use. 
```

Comment:
"ready for use" might not be fully accurate or reflective of what will happen. I think removing this line is fine too or we can say the Sector is now ready to be added to a miner's claimed power. See https://github.com/filecoin-project/specs-actors/blob/eac38c1d153cb985fa147c97113b28718361c90b/actors/builtin/power/power_actor.go#L194

Reply:
Changed the wording, let me know if that works or if it'd be better to remove it entirely :)